tools/autodeployment/deploy.sh

```diff
@@ -1,30 +1,24 @@
-#!/bin/bash
-set -e
-set -u
-set -x
+#!/bin/bash -e -u -x
```

Comment:
Interesting, I haven't seen this done on the shebang before.

Reply:
What's interested is I've never seen it done the inverse way haha.

Reply:
Reverting since the version of BASH on Fedora doesn't enjoy this syntax.